R/WorkbookClass.R

```diff
@@ -686,6 +693,46 @@ Workbook$methods(
 
       .self$writeDrawingVML(xldrawingsDir)
     }
+    
+    ## Threaded Comments xl/threadedComments/threadedComment.xml
+    if (nThreadComment > 0){
+      xlThreadComment <- file.path(tmpDir, "xl", "threadedComments")
+      dir.create(path = xlThreadComment, recursive = TRUE)
+      
+      for (i in 1:nSheets) {
+        if (length(threadComment[[i]]) > 0) {
+          fl <- threadComment[[i]]
+          file.copy(
+            from = fl,
+            to = file.path(xlThreadComment, basename(fl)),
+            overwrite = TRUE,
+            copy.date = TRUE
+          )
+
+          worksheets_rels[[i]] <<- unique(c(
+            worksheets_rels[[i]],
+            sprintf(
+              '<Relationship Id="rIdthread" Type="http://schemas.microsoft.com/office/2017/10/relationships/threadedComment" Target="../threadedComments/%s"/>',
```

Comment:
I'm not entirely convinced that not providing a precise id is a working solution, as I suspect this to be a possible cause of corruption (might work though). It's like this for comments and for vml, so :shrug:

Reply:
I had exactly the same concerns but decided to use the same logic as `comments` because it was easier. 
I agree, it is not really clean and might need to be cleaned along with comments and vml.

From what I have seen Microsoft Office only complains if there are duplicated `Relationship Id`in an xml file. It doesn't seems to care about the value itself.